runhouse/servers/cluster_servlet.py

```diff
@@ -306,20 +306,23 @@ async def aperiodic_cluster_checks(self):
                     if logs_resp is None:
                         # Failed to build the logs payload to send to Den
                         break
-
-                    logs_status_code = logs_resp.status_code
-                    if logs_status_code != 200:
-                        logger.error(
-                            f"{logs_status_code}: Failed to send cluster logs to Den: {logs_resp.josn()}"
-                        )
+                    elif logs_resp == "No logs were sent to den":
```

Comment:
don't love that this function is returning so many values, and a little to follow potential breakage. 

can we move the actual call to Den out of that functoon, so that logs_resp is only related to the actual building of hte logs themselves, and not also tied to sending the logs to den? 


( I mean taking this out of the function entirely: 

```
  post_logs_resp = requests.post(
                    f"{api_server_url}/resource/{cluster_uri}/logs",
                    data=json.dumps(logs_data),
                    headers=rns_client.request_headers(),
                )
```
)

Reply:
yeah, good point